src/SonarScanner.MSBuild.Shim/AdditionalFilesService.cs

```diff
@@ -113,23 +110,27 @@ private static AdditionalFiles PartitionAdditionalFiles(List<string> allFiles, A
     }
 
     private static string[] GetTestExtensions(AnalysisProperties properties) =>
-        properties
-            .Where(x => SupportedTestLanguages.Contains(x.Id))
-            .SelectMany(x => x.Value.Split(Comma))
-            .SelectMany(x => SupportedTestInfixes.Select(infix => $"{infix}{x.Trim()}$"))
-            .Distinct()
-            .ToArray();
+        properties is null
```

Comment:
I did not check for nullability here on the previous PR, so I fixed it on this one.